docs/Dokumentacja Funkcji/MusicPlayer.js.md

```diff
@@ -1,30 +1,236 @@
+}
+```
+
+W przypadku gdy funkcja wywali błąd to zwraca log z informacją o tym który wygląda tak:
+!["Jak wygląda log z funkcji getPlayingSong gdy funkcja wywali błąd"](https://i.imgur.com/6jIL9Xt.png)
+
+W trybie debugowania jeżeli funkcja przekroczy czas bramki to zwraca log z informacją o tym który wygląda tak:
+
+!["Jak wygląda log z funkcji getPlayingSong gdy funkcja przekroczy limit czasowy w trybie debugowania"](https://i.imgur.com/lof8Ku0.png)
+
+W przypadku błędu błąd zostaje zapisany przez [`DebugSaveToFile()`](https://github.com/PFilip08/elektron-radio-player/blob/master/docs/Dokumentacja%20Funkcji/DebugMode.js.md#debugsavetofile) w lokalizacji ``debug/MusicPlayer/getPlayingSong/catched_error.txt`` i potwierdzone to zostaje następującym logiem:
+
+!["Jak wygląda log z funkcji getPlayingSong mówiący o zapisaniu błędu do pliku w trybie debugowania"](https://i.imgur.com/Wsf9M6I.png)
+
+# playlistSongQuery()
+
+Funkcja przyjmuje jeden argument. którym jest `playlistID` jak sama nazwa wskazuje jest to id playlisty z której chcemy odczytać jakie piosenki się znajdują na podanej playliście.
+Na początku funkcja mapuje wszystkie przefiltrowane pliki mp3 z podanej ścieżki i potem zakłada na nie obietnice które zwracają następujące dane w formacie object:
+
+```object
+{
+  title: 'Nazwa piosenki',
+  artist: 'Nazwa artysty',
+  coverData: 'Dane okładki'
+}
+W przypadku gdy nie znajdzie tytułu to object wygląda tak:
+```object
+{
+  title: 'Nazwa Pliku',
+  artist: 'Brak artysty',
+  coverData: 'Brak okładki'
+}
+```
+
+W przypadku gdy nie znajdzie autora to object wygląda tak:
+
+```object
+{
+  title: 'Nazwa Piosenki',
+  artist: 'Nieznany Artysta',
+  coverData: 'Dane okładki'
+}
+```
+
+W przypadku gdy nie znajdzie okładki to object wygląda tak:
+
+```object
+{
+  title: 'Nazwa Piosenki',
+  artist: 'Nazwa Artysty',
+  coverData: 'taboret'
+}
+```
+
+W przypadku gdy wywali błąd to zwraca log z informacją o tym który wygląda tak:
+!["Jak wygląda log z funkcji playlistSongQuery gdy funkcja wywali błąd"](https://i.imgur.com/xrhKfhi.png)
+I zwraca object:
+
+```object
+{
+  title: 'Nazwa Pliku',
+  artist: 'Nieznany Artysta',
+}
+```
+
+## Tryb Debugowania
+
+W przypadku błędu błąd zostaje zapisany przez [`DebugSaveToFile()`](https://github.com/PFilip08/elektron-radio-player/blob/master/docs/Dokumentacja%20Funkcji/DebugMode.js.md#debugsavetofile) w lokalizacji ``debug/MusicPlayer/playlistSongQuery/catched_error.txt`` i potwierdzone to zostaje następującym logiem:
+
+!["Jak wygląda log z funkcji playlistSongQuery mówiący o zapisaniu błędu do pliku w trybie debugowania"](https://i.imgur.com/ct5esYk.png)
+
+# playlistListQuery()
+
+Funkcja zwraca listę playlist w formacie array:
+
+```array
+[
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6'
+]
+```
+
+Robi to poprzez zwykłe wylistowanie całego katalogu `mp3/` i sprawdzeniu co jest folderem i sprawdzeniu czy folder nie nazywa się `onDemand` i zwraca object z nazwą pliku oraz jego id.
```

Comment:
"katalogu `mp3/` i sprawdzeniu" - zamiast "i" przecinek.
"folderem i sprawdzeniu czy" - to samo i przecinek przy "czy"

Reply:
> "katalogu mp3/ i sprawdzeniu" - zamiast "i" przecinek.
Nie zgadzam się z tym stwierdzeniem bo czemu? Bez i to dziwnie brzmi

Reply:
no ok